src/components/LocationSelectorDialog.vue

```diff
@@ -77,7 +77,7 @@
               </h3>
               <v-row v-if="results.length">
                 <v-col cols="12" sm="6">
-                  <LocationSearchResultCard v-for="(location, index) in results" :key="index" :location="location" :hideLocationFooterRow="true" class="mb-2" width="100%" elevation="1" @click="selectLocation(location)" />
+                  <LocationCard v-for="(location, index) in results" :key="index" :location="location" :hideLocationFooterRow="true" class="mb-2" width="100%" elevation="1" @click="selectLocation(location)" />
```

Comment:
forgot `:readonly="true"`, fixed in https://github.com/openfoodfacts/open-prices-frontend/commit/ec311c157f185d9eda72484b2dfff34a1d3f2ea5